build.gradle

```diff
@@ -85,10 +85,10 @@ dependencies {
     // Terra libraries
     implementation group: 'bio.terra', name: 'terra-common-lib', version: '0.0.76-SNAPSHOT'
     implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: '1.2.4-SNAPSHOT'
-    implementation group: 'bio.terra', name: 'workspace-manager-client', version: '0.254.345-SNAPSHOT'
+    implementation group: 'bio.terra', name: 'workspace-manager-client', version: '0.254.575-SNAPSHOT'
 
     // hk2 is required to use WSM client, but not correctly exposed by the client
-    implementation group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: '3.0.8'
+    implementation group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: '2.35'
```

Comment:
shouldn't it be 2.32 if you are matching?
https://github.com/DataBiosphere/terra-workspace-manager/blob/433c236206ded0e101708c5adfb96f34ad0cd8bf/client/gradle/dependencies.gradle#L3

Reply:
ah you are matching the 2.35 below.

can we define a jerseyVersion variable and share it to prevent future discrepency?